docs/user/user/adding-glean-to-your-project/python.md

```diff
@@ -24,6 +24,16 @@ $ python -m pip install glean_sdk
 
 ##### Important
 
+> Installing Python wheels is still a rapidly evolving feature of the Python package ecosystem.
+> If the above command fails, try upgrading `pip`:
+>  ```bash
+>  $ python -m pip install --upgrade pip
+>  ```
```

Comment:
```suggestion
> ```bash
> python -m pip install --upgrade pip
> ```
```

Loose the `$`, makes it easier to copy'n'paste.
(also fixed the indentation, no need for that extra space I think)